### DISCUSSION TITLE
Allow Auto Links to scan more areas

### DISCUSSION THREAD

**Discussion by stevejburge** [publishpress/publishpress-taxonomies#1547] [Feature Requests]
Similar to https://github.com/TaxoPress/TaxoPress/discussions/1492

> I was wondering if taxopress will be able to add links automatically to my term descriptions.

**Comment by stevejburge** [comment]
One request for Permalinks

**Comment by ojopaul** [comment]
@stevejburge Autolink doesn't automatically add links by modifying post when saved/saving to database but instead, we modify output before they're displayed using filters like the_content for example. 

The reason for this is as not to corrupt saved content since anything could go wrong with autolink and even once auto link, there won't be ways to remove the link incase the terms changes.

Now, I'm wondering how this will work with term description since different themes uses function like https://developer.wordpress.org/reference/functions/term_description/ to get terms description which doesn't have any filter and no standard way to filter term description unlike content

**Comment by stevejburge** [comment]
@ojopaul OK, thanks. Let's leave this one open until we get more user feedback
